responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -14,18 +14,14 @@ dependencies:
   - cpuonly
   - pip:
     - numpy==1.21.6
-    - responsibleai~=0.24.1
-    - raiwidgets~=0.24.1
+    - responsibleai~=0.26.0
```

Comment:
I think responsibleai_text and responsibleai_vision packages will also need to be updated then to make this image actually work still, hmm